Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.yml

```diff
@@ -0,0 +1,521 @@
+      required: true
+      secret: false
+    deprecated: false
+    description: Delete deceptive servers
+    execution: false
+    name: illusive-delete-deceptive-servers
+  - arguments:
+    - default: false
+      description: Policy name to assign
+      isArray: false
+      name: policy_name
+      required: true
+      secret: false
+    - default: false
+      description: |-
+        List of hosts to assign, in the following format: machine@domain.
+        Maximum number of hosts is 1000.
+      isArray: false
+      name: hosts
+      required: true
+      secret: false
+    deprecated: false
+    description: Assign a deception policy to domain hosts
+    execution: false
+    name: illusive-assign-host-to-policy
+  - arguments:
+    - default: false
+      description: |-
+        List of hosts to remove policy assignment from, in the following format: machine@domain.
+        Maximum number of hosts is 1000
+      isArray: false
+      name: hosts
+      required: true
+      secret: false
+    deprecated: false
+    description: Remove deception policy assignment from domain hosts
+    execution: false
+    name: illusive-remove-host-from-policy
+  - arguments:
+    - default: false
+      description: The host fqdn or IP address on which to collect forensics
+      isArray: false
+      name: fqdn_or_ip
+      required: true
+      secret: false
+    deprecated: false
+    description: Collect forensics on a specified host and retrieve the forensics
+      timeline
+    execution: false
+    name: illusive-run-forensics-on-demand
+    outputs:
+    - contextPath: Illusive.Event.eventId
+      description: 'The created event ID of the operation '
+      type: String
+  - arguments:
+    - default: false
+      description: |
+        The desired incident ID to retrieve.
+        If specified - other arguments are ignored and only a single incident can be retrieved
+      isArray: false
+      name: incident_id
+      required: false
+      secret: false
+    - default: false
+      description: The list of hostnames to retrieve incidents
+      isArray: true
+      name: hostnames
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether to retrieve incidents with forensics only
+      isArray: false
+      name: has_forensics
+      predefined:
+      - 'True'
+      - ' False'
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        Use offset and limit for pagination.
+        The maximum limit is 100.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: Use offset and limit for pagination.
+      isArray: false
+      name: offset
+      required: false
+      secret: false
+    - default: false
+      description: 'start_date is in Zulu time format, for example: 1993-09-24T17:30:00.000Z.'
+      isArray: false
+      name: start_date
```

Comment:
specific dates are not playbook friendly .
usually when user creates a new playbook in demisto and uses an integration, he uses relative dates.
Similar to what you do in fetch_incidents `3 days ago` ,etc.